saleor/core/postgres.py

```diff
@@ -35,3 +38,69 @@ class NoValidationSearchVector(SearchVector, NoValidationSearchVectorCombinable)
     This class is only safe to use with expressions that do not contain aggregation
     and/or over clause.
     """
+
+
+class FlatSearchVector(Expression):
```

Comment:
Is it a flat search vector or a flat concatenation? If I understand the code correctly, the resulting expression is something like:

```
(setweight(to_tsvector('foo'), 'A')||setweight(to_tsvector('bar'), 'B')||setweight(to_tsvector('baz'), 'C'))
```

Reply:
Yes it is a flat concat, that's a good point I can rename it so it's more general